synapse/lib/storm.py

```diff
@@ -1461,6 +1466,7 @@ def _stormOperSetProp(self, query, oper):
         for form, nodes in formnodes.items():
             props = formprops.get(form)
             if props:
+                [core.reqUserPerm(('node:prop:set', {'form': form, 'prop': prop})) for prop in props.keys()]
```

Comment:
Interesting pattern. It would allocate less memory if `all(...)` instead of `[]`

Also, all is quite a bit faster, for situations where it matters.
```
In [6]: timeit  [foo(x) for x in range(1000)]
89.1 µs ± 3.63 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [7]: timeit  all(foo(x) for x in range(1000))
742 ns ± 22 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```